ui/raidboss/data/03-hw/raid/a1s.txt

```diff
@@ -122,5 +122,5 @@ hideall "--sync--"
 
 ## Enrage
 3000.0 "--sync--" sync /:E49:Oppressor/ window 3000,0
-3000.0 "--sync--" sync /:E49:Oppressor 0\.5/ window 3000,0
-3010.0 "Self-Destruct Enrage" sync /:Oppressor:E49:/
\ No newline at end of file
+3000.0 "--sync--" sync /:E49:Oppressor 0.5/ window 3000,0
```

Comment:
I think this is an incorrect change.  The previous was correct, I think?

Reply:
this is the only line, which oncluded the slash. All others were without it so then all other lines were wrong.
Also when including the slash, the find_missing_translation script reports following: `.\ui\raidboss\data\03-hw\raid\a1s.txt:125 sync ":E49:Oppressor 0\.5"`

Reply:
ok we really need to clarify how to fix this, as this also happends now on a11s as well (with the E.D.D)

Reply:
I do not see this.  When I run this with and without your pull request applied, `python util/find_missing_translations.py -l de -f a1s` does not show any errors in a1s.txt, only a1s.js.